include/graphene-config.h.meson

```diff
@@ -24,7 +24,11 @@ extern "C" {
 #  endif
 
 #  if defined(__GNUC__) && (__GNUC__ >= 5 || (__GNUC__ == 4 && __GNUC_MINOR__ >= 9)) && !defined(__arm__)
-#mesondefine GRAPHENE_HAS_GCC
+#mesondefine GRAPHENE_HAS_INTRINSICS
+#  endif
+
+#  if defined(__clang__)
+#mesondefine GRAPHENE_HAS_INTRINSICS
```

Comment:
`GRAPHENE_HAS_GCC` is an exported public symbol, so it cannot be removed.

You should still define it alongside `GRAPHENE_HAS_INTRINSICS` for the GCC branch.

Reply:
alright